resources/analysisTools/indelCallingWorkflow/TiN_CanopyBasedClustering.R

```diff
@@ -61,7 +61,8 @@ source(opt$cFunction)
 ##### Data Analysis
 # read the Rare.txt file and chromosome left file
 dat<-read.delim(opt$file, header=T, sep="\t")
-chr.length <- read.table(opt$chrLength, header=T)
+chr.length <- read_tsv(opt$chrLength, col_names=c("CHR", "Length"))
+chr.length$shiftLength <- c(0, chr.length$Length[1:23])
```

Comment:
What is the purpose of this `shiftLength`, here? I don't see it used anywhere in this script, but it is a column in `resources/analysisTools/indelCallingWorkflow/chrlengths.plain.sorted.txt` -- otherwise I don't see any references to the name (could be it is used by column index somewhere else, though).

Is it correct, that it is just the length of the previous chromosome in the list? Or should it be the cumulative length of all chromosomes up to that point?